lessons/introR/lesson.Rmd

```diff
@@ -75,7 +76,7 @@ On the top of each `.Rmd` file is the
 [YAML](https://en.wikipedia.org/wiki/YAML) front matter, which looks
 like:
 
-```
+~~~
```

Comment:
Is this necessary? It appears that the existing pages is formatted correctly, no?